File: c#/highLow.cs
        public static string HighLow(string str) 
        /* Create a function that accepts a string of space separated numbers and returns the highest and lowest number (as a string). */
        {
            string[] stringArrayNum = str.Split(' ');

            List<int> numList = new List<int>();

            foreach (string stringomode in stringArrayNum) {
                numList.Add(int.Parse(stringomode));
                


            }

            int[] numArray = numList.ToArray();
            string max = numArray.Max().ToString();
            string min = numArray.Min().ToString();
            return max + " " + min;

        
        }

File: python/highLow.py
'''

Create a function that accepts a string of space separated integers and returns the highest and lowest integers (as a string).
Examples

high_low("1 2 3 4 5") ➞ "5 1"

high_low("1 2 -3 4 5") ➞ "5 -3"

high_low("1 9 3 4 -5") ➞ "9 -5"

high_low("13") ➞ "13 13"

'''


def high_low(txt):
    ans = []

    nums = txt.split()

    for num in nums:
        ans.append(int(num))
    return "{} {}".format(max(ans), min(ans))